inception/inception-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/sidebar/AnnotationSidebarState.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt 
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *  
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.clarin.webanno.ui.annotation.sidebar;
+
+public class AnnotationSidebarState
+{
+    private String selectedTab;
+    private boolean expanded;
+
+    public void setSelectedTab(String aFactoryId)
```

Comment:
Why aFactoryId

Reply:
The id of the AnnotationSidebarFactory that created the selected sidebar. We cannot remember the index of the open tab because that might become invalid e.g. if a user disables/enables a feature flag that causes sidebars to (dis)appear.

Reply:
Thank your for the explanation. I found it just strange that the getter variable and the member variable do not have the same name.